web_advanced_filter/static/src/xml/web_advanced_filter.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="AdvancedFilter">
+        <div role="separator" class="dropdown-divider"/>
+        <button type="button" class="dropdown-item o_advanced_filter o_closed_menu" aria-expanded="false">Whole selection (criteria)</button>
+        <div class="dropdown-item-text o_advanced_filter-domain">
+            <button type="button" class="dropdown-item o_advanced_filter-add-new">To new filter</button>
+            <button type="button" class="dropdown-item o_advanced_filter-add-existing">To existing filter</button>
+            <button type="button" class="dropdown-item o_advanced_filter-remove">Remove from existing filter</button>
```

Comment:
This wording confused me at first.
Instead of "To existing filter" I think I'd write "Add to existing filter", to differentiate it from "To new filter" and make it clear that you're not replacing anything.
I first interpreted "Remove from existing filter" as something you'd use to remove a helper filter that's already part of the filter. Maybe "Subtract from existing filter" would be better. Or another word instead of "subtract".